def foo() -> void:
    x: int
    x: int  # Error: a variable named 'x' already exists
